### DISCUSSION TITLE
SEB with seemles autostart on W10

### DISCUSSION THREAD

**Discussion by kmh42** [SafeExamBrowser/seb-win-refactoring#660] [Q&A]
I have SEB already in autostart, but it takes a while till it is starting. I would like to have the user/student not see the windows desktop when starting up but immediately the SEB session. Is there a way to configure this?

Any help would be appreciated.

**Comment by dbuechel** [comment]
There are a few options, but you'd need to research the details online. You most likely should be able to configure Windows via the registry and/or group policies, and there also is an option to specify a custom shell, but I do neither have any experience with the latter nor do I know the details.
